### DISCUSSION TITLE
Does renovate/artifacts only show up in GitHub checks if fails?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [renovatebot/renovate#10215] [Request Help] [answered]
I am trying to update my [mergify](https://mergify.io/) rules to avoid auto merge if renovate/artifacts fails like this: https://github.com/Hongbo-Miao/hongbomiao.com/pull/2459#issuecomment-850382330

I saw this "renovate/artifacts" check in this pull request:
![image](https://user-images.githubusercontent.com/3375461/120009902-14be7280-c00f-11eb-81b6-570965d9c9b3.png)

I want to confirm: is this "renovate/artifacts" only shows if renovate/artifacts fails?
Because in other pull requests opened by Renovate which has no artifact errors, they do not have this GitHub check, such as
https://github.com/Hongbo-Miao/hongbomiao.com/pull/2458
![image](https://user-images.githubusercontent.com/3375461/120010004-361f5e80-c00f-11eb-9e6e-0f14406c37cd.png)

Thanks!

**Comment by viceice** [comment, marked as answer]
yes, it will only added if artifacts are failing

**Reply by hongbo-miao** [reply]
Thanks for confirming, @viceice !

**Reply by hongbo-miao** [reply]
For future people who ends up here,  the correct Mergify rule for this case is `check-failure!=renovate/artifacts`.
I posted at https://github.com/Mergifyio/mergify-engine/discussions/2698#discussioncomment-800130
